src/test/java/md/miller1995/Dealership/repositories/CarRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package md.miller1995.Dealership.repositories;
+
+import md.miller1995.Dealership.models.entities.CarEntity;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please keep an empty row between the last import and the class or the annotations of the class.

Reply:
- add empty row  between the last import and the annotation